platform/javascript/os_javascript.cpp

```diff
@@ -415,12 +415,134 @@ void OS_JavaScript::set_cursor_shape(CursorShape p_shape) {
 
 	ERR_FAIL_INDEX(p_shape, CURSOR_MAX);
 
+	if (p_shape == cursor_shape) {
+		return;
+	}
```

Comment:
Seems simpler to remove this conditional return instead of the `cursor_shape = CURSOR_MAX` magic

Reply:
Removed.